src/app/api/volunteer/auth/password-reset/create/route.ts

```diff
@@ -0,0 +1,37 @@
+import { createPasswordReset } from "@server/mongodb/actions/PasswordReset";
+import { getUserByEmail } from "@server/mongodb/actions/User";
+import APIWrapper from "@server/utils/APIWrapper";
+
+type RequestData = {
+  email: string;
+  name: string;
+};
+
+export const POST = APIWrapper({
+  config: {
+    requireToken: true,
```

Comment:
This route should not require a token since the user will not be signed in for password reset.